converters/toZigbee.js

```diff
@@ -5373,6 +5373,7 @@ const converters = {
         key: ['local_temperature_calibration'],
```

Comment:
Its all about local_temperature_calibration, it should accept two digs resolution to match the local temp resolution, on devices like the Eurotronics Spirit it just works fine, but the tuya devices have this hex convertion thing. Its a little bit strange, i have saswell thermostats sometimes the value is rounded without any error, somtimes it throwes errors like "RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 255. Received 27_487_790_694" i think the hex converter should be able to convert 0.1 values as well or round it at least to prevent errors.

Reply:
I guess it should be integer value with some precision.
For example, you can push `0.34` to the MQTT, but convert it to the integer with same precision as dp was configured on the device. For temperature I think it can be something like `100`, but need to sniff this value from the original app.

So, you should try something like `Math.round(value/100)` inside `local_temperature_calibration` function and pass integer value to the datapoint.

Reply:
Sry i didn't understand your point, if you make a Math.round(value/100) on 0.34 or 2.3 it will be 0.

As an example, the saswell local_temperature is 18.8°C so if you want to calibrate it to 20°C the calibration must be 0.2

Reply:
Sorry, I mean `Math.round(value*100)` of cause.

Reply:
But I'm not sure that precision of this datapoint is exactly `100`. You should check it from the device docs or from the sniffer.

Reply:
OK, yes i will investigate it, but i'm confused by the Hex 4 Bite stuff, maybe someone with more knowledge about tuya devices can write a proper converter, i tried the Math.round(value*10) this shoud be the right resolution, but get an range overflow to 4294967295 anyway